### PR TITLE
Docs: moved core dump instructions to install guide

### PR DESCRIPTION
**Pull request by mperezfuster**
Ref https://github.com/greenplum-db/gpdb/issues/12329
Core dump instructions should be part of the Install Guide rather than Best Practices.
Preview page:
https://mireia-core.sc2-04-pcf1-apps.oc.vmware.com/7-0/install_guide/prep_os.html#topic3__core_dump

### CODE REVIEW

File: gpdb-doc/dita/install_guide/prep_os.xml [hunk 1]
```diff
@@ -292,56 +292,67 @@ vm.dirty_ratio = 10</codeblock></p>
...
+        <p>Enable core files output to a known location by adding the following line to
+            <codeph>/etc/sysctl.conf</codeph>:<codeblock>kernel.core_pattern=/var/core/core.%h.%t
+</codeblock></p>
+      </section>
+      <p>Run the following command to make the change to the live
+        kernel:<codeblock># sysctl -p</codeblock></p>
+      <p>Make sure <codeph>limits.conf</codeph> allows core
+        files.<codeblock># grep core /etc/security/limits.conf  
+* soft  core unlimited</codeblock></p>
+      <section id="xfs_mount">
```
Comment (lenapivotal) [inline gpdb-doc/dita/install_guide/prep_os.xml:325]: I think there's an extra space between soft and core. 
Also, we're asking them to ensure that "soft core unlimited" exists but is it in the `limits.conf` file by default? OR do they add it? Is it a file that user can add various entries?

File: gpdb-doc/dita/install_guide/prep_os.xml [hunk 2]
```diff
@@ -292,56 +292,67 @@ vm.dirty_ratio = 10</codeblock></p>
...
+          131072.</p>
+      </section>
+      <section id="core_dump">
+        <title>Core Dump</title>
+        <p>Enable core files output to a known location by adding the following line to
+            <codeph>/etc/sysctl.conf</codeph>:<codeblock>kernel.core_pattern=/var/core/core.%h.%t
+</codeblock></p>
+      </section>
+      <p>Run the following command to make the change to the live
+        kernel:<codeblock># sysctl -p</codeblock></p>
```
Comment (lenapivotal) [inline gpdb-doc/dita/install_guide/prep_os.xml:321]: Maybe:
To enable the change to the live kernel, run the following command:

**Review by lenapivotal — APPROVED** [review]
Small suggestions that can be ignored.

File: gpdb-doc/dita/install_guide/prep_os.xml [hunk 3]
```diff
@@ -292,56 +292,67 @@ vm.dirty_ratio = 10</codeblock></p>
...
+          sets user limits by reading the values from the <codeph>limits.conf</codeph> file and then
+          from the override file. For information about PAM and user limits, see the documentation
+          on PAM and <codeph>pam_limits</codeph>.</p>
+        <p>Run the <codeph>ulimit -u</codeph> command on each segment host to display the maximum
+          number of processes that are available to each user. Validate that the return value is
+          131072.</p>
+      </section>
+      <section id="core_dump">
+        <title>Core Dump</title>
+        <p>Enable core files output to a known location by adding the following line to
```
Comment (VMWareJennyBeth) [inline gpdb-doc/dita/install_guide/prep_os.xml:316]: What is this sentence trying to say? Is it to specify where core files' output should go? Is it to just enable output from core files? Is it the output of the core files?

Comment (mperezfuster) [reply gpdb-doc/dita/install_guide/prep_os.xml:316]: Output acts like a verb here. Since it is not too clear, I will rephrase it.

File: gpdb-doc/dita/install_guide/prep_os.xml (hunk 1 above)
Comment (mperezfuster) [reply gpdb-doc/dita/install_guide/prep_os.xml:325]: Thanks, I will rephrase it to ask user to add the line.

**Review by ashwinstar — APPROVED** [review]

File: gpdb-doc/dita/install_guide/prep_os.xml [hunk 4]
```diff
@@ -292,56 +292,66 @@ vm.dirty_ratio = 10</codeblock></p>
...
+        <p>Run the <codeph>ulimit -u</codeph> command on each segment host to display the maximum
+          number of processes that are available to each user. Validate that the return value is
+          131072.</p>
+      </section>
+      <section id="core_dump">
+        <title>Core Dump</title>
+        <p>Enable core file generation to a known location by adding the following line to
+            <codeph>/etc/sysctl.conf</codeph>:<codeblock>kernel.core_pattern=/var/core/core.%h.%t
+</codeblock></p>
+      </section>
```
Comment (dyozie) [inline gpdb-doc/dita/install_guide/prep_os.xml:319]: This closing section tag seems out of place.  The paragraphs that follow still seem to apply to the core dump section.  Either that, or they're orphaned and not part of a section.

**Review by dyozie — CHANGES_REQUESTED** [review]

**Review by dyozie — APPROVED** [review]
